packages/auth/src/auth-service.ts

```diff
@@ -58,18 +58,7 @@ export class AuthService {
    * @param role - New wso2 role
    * @returns
    */
-  async assignWSO2UserRole(email: string, role: string) {
-    return requests.assignWSO2UserRole(email, role)
-  }
-
-  /**
-   *
-   * @param userId - wso2 role to remove
-   * @param userName - wso2 username
-   * @param role - wso2 role
-   * @returns
-   */
-  async removeWSO2UserRole(email: string, role: string) {
-    return requests.removeWSO2UserRole(email, role)
+  async updateWSO2UserRole(email: string, role: string, operation: 'add' | 'remove') {
```

Comment:
Let's leverage the new enum here - notice how it reads better when it is singular in this context:
`async updateWSO2UserRole(email: string, role: string, operation: RoleOperation)`